config/dispatcher/dispatcher.ex

```diff
@@ -313,6 +317,10 @@ defmodule Dispatcher do
     forward conn, path, "http://sync-ovo-numbers/sync-from-kbo/"
   end
 
+  post "/kbo-data-sync/*path", %{ layer: :api_services, accept: %{ json: true } } do
+    forward conn, path, "http://kbo-data-sync/sync-kbo-data/"
+  end
+
```

Comment:
**Remark:** Same remark as my other one, make sure to configure the correct route for the final version of the updated service and remove the unused one.